Podfile

```diff
@@ -1,113 +0,0 @@
-source 'https://cdn.cocoapods.org/'
```

Comment:
Should update the Sequel Ace readme to drop the `bundle exec pod install --repo-update` line! :)

Reply:
Riiight!